gms/api/src/main/pegasus/com/linkedin/dataprocess/DataProcess.pdl

```diff
@@ -0,0 +1,34 @@
+namespace com.linkedin.dataprocess
+
+import com.linkedin.common.ChangeAuditStamps
+import com.linkedin.common.DataProcessUrn
+import com.linkedin.common.DatasetUrn
+import com.linkedin.common.Uri
+import com.linkedin.common.VersionTag
+
+/**
+ * Spec. for a process. A collection of data conforming to a single schema that can evolve over time.
+ */
+record DataProcess includes DataProcessKey, ChangeAuditStamps, VersionTag {
+
+  /**
+   * Process unique identifier. System assigned value when a new Process is created.
+   */
+  id: long = 0
```

Comment:
The `Value` model of the top-level rest.li endpoint should include only the key & aspects. As such you don't need `id`, `urn` and `uri` here.

Reply:
I am absolutely not familiar with rest.li. I copied `Dataset` s definition. Wonder why it has those `id`, `urn` and `uri`. Code has been updated.